app/lib/search/text_utils.dart

```diff
@@ -6,6 +6,7 @@ final RegExp _separators =
     new RegExp(r'[_\.,;=\(\)\>\<\[\]\{\}\|\?\!\/\+\-\*]|\s');
 final RegExp _nonCharacterRegExp = new RegExp('[^a-z0-9]');
 final RegExp _multiWhitespaceRegExp = new RegExp('\\s+');
+final RegExp _exactTermRegExp = new RegExp('"(.*?)"');
```

Comment:
Instead of using a non-greedy operator (which is super hard to spot in the first place), maybe we can use this:

```dart
final RegExp _exactTermRegExp = new RegExp(r'"([^"]+)"')
```